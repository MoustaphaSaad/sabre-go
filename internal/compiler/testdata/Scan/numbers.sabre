x := 42
y := 3.14
z := 0xFF
w := 0b1010
q := 0o777
