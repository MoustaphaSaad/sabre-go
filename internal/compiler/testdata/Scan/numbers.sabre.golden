IDENTIFIER      "x"                     1:1    [0-1]
:=              ":="                    1:3    [2-4]
LITERAL_INT     "42"                    1:6    [5-7]
IDENTIFIER      "y"                     2:2    [8-9]
:=              ":="                    2:4    [10-12]
LITERAL_FLOAT   "3.14"                  2:7    [13-17]
IDENTIFIER      "z"                     3:2    [18-19]
:=              ":="                    3:4    [20-22]
LITERAL_INT     "0xFF"                  3:7    [23-27]
IDENTIFIER      "w"                     4:2    [28-29]
:=              ":="                    4:4    [30-32]
LITERAL_INT     "0b1010"                4:7    [33-39]
IDENTIFIER      "q"                     5:2    [40-41]
:=              ":="                    5:4    [42-44]
LITERAL_INT     "0o777"                 5:7    [45-50]
EOF             ""                      6:2    [51-51]
